blockfrost-api/src/Blockfrost/Types/Cardano/Epochs.hs

```diff
@@ -88,10 +88,32 @@ data ProtocolParams = ProtocolParams
   , _protocolParamsMaxCollateralInputs    :: Integer -- ^ The maximum number of collateral inputs allowed in a transaction
   , _protocolParamsCoinsPerUtxoSize       :: Lovelaces -- ^ The cost per UTxO size. Cost per UTxO *word* for Alozno. Cost per UTxO *byte* for Babbage and later
   , _protocolParamsCoinsPerUtxoWord       :: Lovelaces -- ^ The cost per UTxO word (DEPRECATED)
+  , _protocolParamsPvtMotionNoConfidence :: Maybe Rational
+  , _protocolParamsPvtCommitteeNormal :: Maybe Rational
+  , _protocolParamsPvtCommitteeNoConfidence :: Maybe Rational
+  , _protocolParamsPvtHardForkInitiation :: Maybe Rational
+  , _protocolParamsPvtppSecurityGroup :: Maybe Rational
+  , _protocolParamsDvtMotionNoConfidence :: Maybe Rational
+  , _protocolParamsDvtCommitteeNormal :: Maybe Rational
+  , _protocolParamsDvtCommitteeNoConfidence :: Maybe Rational
+  , _protocolParamsDvtUpdateToConstitution :: Maybe Rational
+  , _protocolParamsDvtHardForkInitiation :: Maybe Rational
+  , _protocolParamsDvtPPNetworkGroup :: Maybe Rational
+  , _protocolParamsDvtPPEconomicGroup :: Maybe Rational
+  , _protocolParamsDvtPPTechnicalGroup :: Maybe Rational
+  , _protocolParamsDvtPPGovGroup :: Maybe Rational
+  , _protocolParamsDvtTreasuryWithdrawal :: Maybe Rational
+  , _protocolParamsCommitteeMinSize :: Maybe Quantity
+  , _protocolParamsCommitteeMaxTermLength :: Maybe Quantity
+  , _protocolParamsGovActionLifetime :: Maybe Quantity
+  , _protocolParamsGovActionDeposit :: Maybe Lovelaces
+  , _protocolParamsDrepDeposit :: Maybe Lovelaces
+  , _protocolParamsDrepActivity :: Maybe Quantity
+  , _protocolParamsMinFeeRefScriptCostPerByte :: Maybe Rational
```

Comment:
This is chosen as `Maybe Rational` instead of `Maybe Integer` as corresponding type defined in cardano-ledger codebase is (loosely put) `Ratio Word64`.